.travis/test_06_script.sh

```diff
@@ -50,7 +51,8 @@ END_FOLD
 
 if [ "$RUN_TESTS" = "true" ]; then
   BEGIN_FOLD unit-tests
-  DOCKER_EXEC LD_LIBRARY_PATH=$TRAVIS_BUILD_DIR/depends/$HOST/lib make $MAKEJOBS check VERBOSE=1
+  echo Running the unit tests with an increased timeout
```

Comment:
We could add some kind of warning about the absence of output, like "keep patience, it could take a while" or smth like that.